breadcord/core_modules/modulemanager/__init__.py

```diff
@@ -69,10 +69,11 @@ async def cog_app_command_error(
             ))
 
     @app_commands.command(description="Install a module from github")
-    @app_commands.describe(module="A github url or repo path")
+    @app_commands.describe(module="A github url or repo path", branch="A branch of the repo")
     @app_commands.check(breadcord.helpers.administrator_check)
-    async def install(self, interaction: discord.Interaction, module: str):
-        module = GH_BASE_URL.sub('', module).removesuffix('.git')
+    async def install(self, interaction: discord.Interaction, module: str, branch: str | None = None):
+        module = GH_BASE_URL.sub('', module).removesuffix('.git').replace(" ", "-")
```

Comment:
Why is this changed?

Reply:
I’m going to assume you mean like 75. 
It was changed to be more in line with how I handle branches, which was done to give more leniency when specifying the name, and GitHub replaces spaces in repo/branch names with dashes upon creation.

Reply:
Alright - I was unsure about the decision to default to hyphens instead of e.g. underscores, but if that's GitHub's default then I'm fine with that. It's probably now deserving of a transformer with autocomplete though

Reply:
Making a transformer sounds suspiciously much like a you issue :trollface: